### PR TITLE
Sparklines for Finance and Growth

### PR DESCRIPTION
**Pull request by belucid**
This PR adds sparklines into the finance and growth sections of emails, and removes the "card" styling in favor of something more minimal.

https://trello.com/c/pBN9SNtb
https://trello.com/c/VzhV4x8T

Depends on this commit to infra (already merged): https://github.com/belucid/open-company-infrastructure/commit/5e94c9648ac639892001d37190f0692b58acf91a

To test:
- Review the code
- Run through the snapshots in `content.clj`'s REPL comment section, looking at `hiccup.html` after each one. Make changes to the JSON between runs, are they reflected in the sparklines? Try to challenge the sparkline code w/ challenging data changes, any problems?
- Send some email updates from staging (already deployed). Check the email in mobile and web email viewers.
- Merge.
- Deploy to beta.
- Upgrade your data plan.

### CODE REVIEW

File: src/oc/email/content.clj [hunk 1]
```diff
@@ -59,58 +54,68 @@
...
-(defn- topic-image [image-url]
-  [:tr
-    [:th
-      [:center
-        [:img {:src image-url}]]]])
+; (defn- topic-image [image-url]
+;   [:tr
+;     [:th {:class "small-12 large-12 columns first last"}
+;       [:center
+;         [:img {:src image-url}]]]])
```
Comment (bago2k4) [inline src/oc/email/content.clj]: did you mean to leave this commented out?

**Review by bago2k4 — CHANGES_REQUESTED** [review]
Also was unable to complete the steps at the end of content.clj:
`(spit "./hiccup.html" (email/html {}))`
i can't find any ns imported as email, email fn in content.clj accept 2 args not one.

I was unable to run all the steps in sparkline.cljs:
`(ChartUtilities/writeChartAsPNG b c 80 26)`
this return an error:

```
oc.email=> (ChartUtilities/writeChartAsPNG b c 80 26)

CompilerException java.lang.RuntimeException: No such namespace: ChartUtilities, compiling:(/private/var/folders/n0/wg92k8zn0d5bn94q0668gmkm0000gp/T/form-init1448313780304294314.clj:1:1)
oc.email=> (require '[[org.jfree.chart ChartFactory ChartUtilities])
```

and if i try to import ChartUtilities like you do in sparkline.clj:

```
oc.email=> (require '[org.jfree.chart ChartUtilities])

FileNotFoundException Could not locate org/jfree/chart/ChartUtilities__init.class or org/jfree/chart/ChartUtilities.clj on classpath.  clojure.lang.RT.load (RT.java:458)
```

**Comment by belucid** [comment]
@bago2k4 

ChartUtilities is a raw Java class, so it needs imported, not required.

In any case, that export routine that was in the `sparkline.clj` comment is now part of the `sparkchart-html` fn. I just wrote it in the REPL first. Didn't mean for you to run through the comments section of `sparkline.clj`.

File: src/oc/email/content.clj (hunk 1 above)
Comment (belucid) [reply src/oc/email/content.clj]: I didn't. It's gone now. Thanks.

**Review by bago2k4 — APPROVED** [review]

**Pull request merged**